src/stylesheets/elements/_buttons.scss

```diff
@@ -28,19 +28,18 @@ $button-stroke: inset 0 0 0 2px;
 .usa-button:visited,
 .usa-button-primary:visited {
   @include font-smoothing;
-  @include margin(0.5em 0.5em 0.5em null);
+  @include typeset($theme-font-family-button, null, 1);
+  @include u-margin-y(0.5)
```

Comment:
Since we're using a top margin model for paragraph text, can this be revisited so it doesn not have a bottom margin?

Reply:
yeah, I'll try that